app/javascript/all_neighbors_system_dashboard_stack.js

```diff
@@ -477,40 +485,204 @@ class AllNeighborsSystemDashboardRTHStack extends AllNeighborsSystemDashboardSta
     super(data, initialState, selector, options)
   }
 
+  getActiveConfig() {
+    // this.data is an array of arrays, the key is in position 0, data in position 1
+    return this.getActiveSet()[1].config
+  }
+
+  getActiveSeries() {
+    // this.data is an array of arrays, the key is in position 0, data in position 1
+    return this.getActiveSet()[1].series
+  }
+
+  getActiveSet() {
+    let activeSet = this.data.filter((d) => {
+      let equal = true
+      if (this.state.projectType != d[0].projectType) {
+        equal = false
+      }
+      if (this.state.countLevel != d[0].countLevel) {
+        equal = false
+      }
+      if (this.state.demographics != d[0].demographics) {
+        equal = false
+      }
+      return equal
+    })
+    return activeSet[0]
+  }
+
+  // var chart = bb.generate({
+  //   data: {
+  //     columns: [
+  //       ["Housed, completed program", 30, 200, 100],
+  //       ["Returns", 10, 20, 15]
+  //     ],
+  //     type: "bar", // for ESM specify as: line()
+  //   },
+  //   axis: {
+  //     rotated: true,
+  //     x: {
+  //       type: "category",
+  //       categories: [
+  //         "Adult only",
+  //         "Adult & Child",
+  //         "Child Only",
+  //       ],
+  //     }
+  //   },
+  //   bindto: "#categoryAxis"
+  // });
+
+  getDataConfig() {
+    const categories = this.getActiveConfig().axis.x.categories
+
+    const columns = this.getActiveConfig().names.map((name, i) => {
+      return this.getColumn(name, i, categories)
+    })
+    return {
+      order: null,
+      columns: columns,
+      type: "bar",
+      colors: this.config.colors,
+      names: this.config.names,
+      labels: {
+        show: true,
+        centered: false,
+        colors: this.config.label_colors,
+        format: (v, id, i, j) => {
+          return d3.format(",")(v);
+        },
+      },
+      stack: {
+        normalize: false,
+      }
+    }
+  }
+
+  // Given an array of categories, and the index of the array in the values set
+  // collect sums for each category in the date range
+  // return an array of [name, category_1_sum, category_2_sum, ...]
+  getColumn(name, index, categories) {
+    let col = [name]
+
+    // find data that overlaps the filter date range
+    const filtered = this.series.filter((n) => {
+      return this.inDateRange(n.date, this.state.dateRange)
+    })
+    const monthly_counts_for_name = filtered.map((s) => {
+      return s.values[index]
+    })
+
+    const monthly_counts = categories.map((_, cat_index) => {
+      // Gather up all of the returns or placements for the given category
+      return d3.sum(monthly_counts_for_name.map((counts) => counts[cat_index]))
+    })
+    return col.concat(monthly_counts)
+  }
+
   getConfig() {
-    const fitLabels = this.fitLabels
+    // const fitLabels = this.fitLabels
```

Comment:
I'm assuming this is staying in to be possibly turned on later

Reply:
Yes, but I think it's safe to remove now.  Thanks.